src/components/DatePicker/DatePicker.tsx

```diff
@@ -327,12 +327,12 @@ const DatePicker = React.forwardRef<TextInputRefType, DatePickerProps>(
 
     if ((ref && !nameFrom) || (refTo && !nameTo)) {
       console.warn(
-        "A `ref` or `refTo` prop was passed but not the equivalent `nameFrom` or `nameTo` prop."
+        "NYPL Reservoir DatePicker: a `ref` or `refTo` prop was passed but not the equivalent `nameFrom` or `nameTo` prop."
       );
     }
     if (onChange && (ref || refTo || nameFrom || nameTo)) {
       console.warn(
-        "React `ref` props were passed and an `onChange` prop as well. Use whichever is best for your app but not both."
+        "NYPL Reservoir DatePicker: React `ref` props were passed and an `onChange` prop as well. Use whichever is best for your app but not both."
```

Comment:
Could be clearer.
```
React `ref` and `onChange` props were both passed. Use…
```

Or should should `ref` be referred to in a singular sense?  `prop` vs `props`?